packages/ui5-test-writer/templates/v4/integration/opaTests.qunit.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Integration tests</title>
+	<meta charset="utf-8">
+
+	<script id="sap-ui-bootstrap"
+			src="/resources/sap-ui-core.js"
+			data-sap-ui-theme='sap_fiori_3'
```

Comment:
Does the theme need to match what's in the app or is that not relevant?

Reply:
I don't know exactly, but if I remove this line the OPA test doesn't run

Reply:
I guess it is more if this should be changed from `data-sap-ui-theme='sap_fiori_3'` to `data-sap-ui-theme='sap_fiori_3_dark'` or `data-sap-ui-theme='sap_horizon'` to match the `webapp/index.html` for example.

In my view it is not that relevant what theme is used for the test.